backend/src/routes/clients.route.ts

```diff
@@ -0,0 +1,69 @@
+import { clientsController } from '../controllers/clients.controller';
+import { Router } from 'express';
+
+/**
+ * @swagger
+ * tags:
+ *   name: Clients
+ *   description: Client API endpoints
+ */
+/**
+ *  @swagger
+ * components:
+ *  schemas:
+ *   BusinessInfo:
+ *    type: object
+ *    required:
+ *     - accessToken
+ *     - refreshToken
+ *     - businessId
+ *     - businessName
+ *     - businessDescription
+ *    properties:
+ *     accessToken:
+ *      type: string
+ *      description: The JWT Access token
+ *     refreshToken:
+ *      type: string
+ *      description: The JWT Refresh token
```

Comment:
These are not values that you should receive in the body. They are provided by the auth header